src/test/librbd/test_librbd.cc

```diff
@@ -6928,6 +6930,7 @@ TEST_F(TestLibRBD, Migration) {
   rbd_migration_status_cleanup(&status);
 
   ASSERT_EQ(-EBUSY, rbd_remove(ioctx, name.c_str()));
+  ASSERT_EQ(-EINVAL, rbd_trash_move(ioctx, name.c_str(), 0));
```

Comment:
Need to rebased here (and [L6996](https://github.com/ceph/ceph/pull/25529/files#diff-72e3f904206c591a1086f3530472c6d9R6996)) if commit: 6257065cfde83957b2c71f9926a8de0f888c8141 merged first.